ftw/subsite/content/subsitedx.py

```diff
@@ -60,14 +61,12 @@ class ISubsiteSchema(model.Schema):
         required=False,
         missing_value=''
     )
-
+    widget('language_references', ReferenceBrowserWidget,
+           override=True, selectable=["ftw.subsite.Subsite"])
     language_references = RelationList(
         title=_(u'label_language_references', default=u'Languages'),
         default=[],
         missing_value=[],
-        value_type=RelationChoice(title=u'Related Subsites',
-                                  source=SubsiteLanguagObjPathSourceBinder()),
```

Comment:
Is the class `SubsiteLanguagObjPathSourceBinder` used elsewhere or could it be removed altogether?